### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rfenrich**
Clarified a few print statements, corrected some function calls, and fixed the compressive limit state definition for the composite plate example.

### CODE REVIEW

File: grama/eval_random.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def eval_sinews(
         >>> df_design >> gr.pt_auto()
         >>> # Vis results
         >>> df_sinew = md >> gr.ev_sinews(df_det="nom")
-        >>> df_sinew >> gr.pt_auto()
+        >>> df_sinew >> gr.plt_auto()
```
Comment (zdelrosario) [inline grama/eval_random.py]: Pipe-enabled functions use two-letter prefixes; the original code example should be maintained:

```suggestion
        >>> df_sinew >> gr.pt_auto()
```

File: grama/models/plane_laminate.py [hunk 2]
```diff
@@ -226,16 +226,16 @@ def uniaxial_stress_limit(X):
...
     ## Evaluate stress [\sigma_11, \sigma_22, \sigma_12]_i
     Stresses = Nx * uniaxial_stresses(Param, Theta, T)
 
     ## Construct limit state
     g_limit = np.zeros((k, 5))
-    g_limit[:, (0,1)] = +Sigma_max[:, (0,1)] - Stresses[:, (0,1)]
-    g_limit[:, (2,3)] = +Sigma_max[:, (2,3)] + Stresses[:, (0,1)]
-    g_limit[:, 4]     = +Sigma_max[:, 4]     - np.abs(Stresses[:, 2])
+    g_limit[:, (0,1)] = +1 - Stresses[:, (0,1)] / Sigma_max[:, (0,1)]
+    g_limit[:, (2,3)] = +1 - Stresses[:, (0,1)] / Sigma_max[:, (2,3)] # since sig_11_c, sig_22_c > 0
```
Comment (zdelrosario) [inline grama/models/plane_laminate.py]: I believe the original polarity is correct; the max stress criterion for tension and compression (with the compressive strength coded as a positive variable) is:

`-strength_compression < sigma_applied < +strength_tension`

Re-arranging the first inequality yields a limit state for compression

`0 < +strength_compression + sigma_applied = g_compression`

I do like the idea of normalizing the limit states though; that would give the following:

```suggestion
    g_limit[:, (2,3)] = +1 + Stresses[:, (0,1)] / Sigma_max[:, (2,3)]
```

Let me know if the derivation above seems incorrect.

File: grama/eval_random.py [hunk 3]
```diff
@@ -151,7 +151,7 @@ def eval_lhs(
 
     ## Ensure sample count is int
     if not isinstance(n, Integral):
-        print("eval_monte_carlo() is rounding n...")
+        print("eval_lhs() is rounding n...")
```
Comment (zdelrosario) [inline grama/eval_random.py]: Good catch!

File: grama/models/plane_laminate.py [hunk 4]
```diff
@@ -462,6 +462,6 @@ def mapSign(x):
     stress_uniaxial = uniaxial_stresses(Param, Theta, T)
 
     ## Test model
-    model = model_composite_plate_tension(
+    model = make_composite_plate_tension(
```
Comment (zdelrosario) [inline grama/models/plane_laminate.py]: Good catch!

File: grama/models/plane_laminate.py (hunk 2 above)
Comment (rfenrich) [reply grama/models/plane_laminate.py]: Your suggestion is correct. I was viewing the problem from a design perspective of analyzing a laminate subject to tensile and compressive loads. My suggested change is suitable if we consider the compressive limit states for a compressive load. However, we are considering all (tensile and compressive) limit states under a tensile load instead. Naturally a compressive failure will never occur under a tensile load, at least for most simple laminates under simple loading conditions.

**Pull request merged**